builtin/settingtypes.txt

```diff
@@ -404,11 +404,11 @@ shader_path (Shader path) path
 
 [****Advanced Lighting]
 
-#    Enables advanced lighting effects such directional environment lighting
-enable_advanced_lighting (Advanced lighting) bool false
+#    Enables directional environment lighting
+directional_shading (Directional shading) bool true
```

Comment:
There's no room to add an option, and there *should* be no reason to disable it anyway.